ui/components/multichain/import-tokens-modal/index.scss

```diff
@@ -1,3 +1,52 @@
+.scrollable {
+  ::-webkit-scrollbar {
+    display: none; /* for Chrome, Safari, and Opera */
+  }
+}
+
+.banner {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.contentDialogClass {
+  padding-left: 0 !important;
```

Comment:
Whenever we use `!important`, we should add a comment saying why.  Also, this className is too generic, and should relate to the component via BEM.